doc/integrator/ngeo.rst.tmpl

```diff
@@ -117,3 +99,14 @@ You also have a ``default`` entry from where we can give the base values for ``c
 The possible values are described in the ngeo documentation
 `ngeo constants <https://camptocamp.github.io/ngeo/${MAIN_BRANCH}/jsdoc/module-src_options.html>`_,
 `gmf constants <https://camptocamp.github.io/ngeo/${MAIN_BRANCH}/jsdoc/module-contribs_gmf_src_options.html>`_.
+
+CSS style
+---------
+
+In the ``gmfOptions`` we have a ``cssVars`` property where we can configure the CSS variable
+(``var(--a-variable)`` in the css files.
+
+On this dictionary we have some particular variables named ``brand-primary`` and ``brand-secondary``,
+thoses variable configure the major application colors.
+
+You also have an ``<interface>.css`` file to overide the css rules.
```

Comment:
Do we mean:
"You can in addition define an interface.css file to override the CSS rules"
?

Reply:
+1